gramps/plugins/lib/libgedcom.py

```diff
@@ -1129,14 +1129,12 @@ def __init__(self):
 
         try:
             filepath = os.path.join(DATA_DIR, "gedcom.xml")
-            ged_file = open(filepath, "rb")
+            with open(filepath, "rb") as ged_file:
+                parser = GedInfoParser(self)
```

Comment:
I'd go with @QuLogic and we can test it to make sure that works well. I hate general try/except code with silently handling whatever errors may have happened. Thanks!